docs/conf.py

```diff
@@ -19,7 +19,6 @@
 # Mock imports that we don't want to install for builds
 
 autodoc_mock_imports = [
-    'pysmurf.client.util.pub',
```

Comment:
I think it builds, but breaks the docstrings for those functions. With the mock, when you do
```
from pysmurf.client.util.pub import set_action
```
`set_action` becomes a mock function that overwrites the docstring of any funcs that it decorates. I think this can be fixed by changing the `squid_curve` module to use `sodetlib.set_action` which cleanly handles this instead of importing directly from pysmurf.

Reply:
Ah, I see, I didn't realize that mock was causing problems for decorated functions.

Sounds like you have a solution in mind. I think you might also be able to protect it like we used to in OCS:
```
on_rtd = os.environ.get('READTHEDOCS') == 'True'
if not on_rtd:
    from pysmurf.client.util.pub import set_action
```

That might only be helpful when we move to readthedocs though, and requires setting that variable locally when building though, so not as clean as your solution.